nacl-fast.js

```diff
@@ -2361,17 +2361,16 @@ nacl.setPRNG = function(fn) {
   // Initialize PRNG if environment provides CSPRNG.
   // If not, methods calling randombytes will throw.
   var crypto;
-  if (typeof window !== 'undefined') {
-    // Browser.
-    if (window.crypto && window.crypto.getRandomValues) {
-      crypto = window.crypto; // Standard
-    } else if (window.msCrypto && window.msCrypto.getRandomValues) {
-      crypto = window.msCrypto; // Internet Explorer 11+
-    }
-    if (crypto) {
+  if (typeof process === 'undefined' || process.browser) {
```

Comment:
OK, here it is:

https://github.com/dchest/tweetnacl-js/pull/90/files

```
var crypto = typeof self !== 'undefined' ? (self.crypto || self.msCrypto) : null;
if (crypto && crypto.getRandomValues) {
...
  } else if (typeof require !== 'undefined') {
```

If we have self.crypto.getRandomValues, use it, otherwise try require('crypto')